cmd/vulcan-host-discovery/main.go

```diff
@@ -87,13 +89,13 @@ func discoveredHosts(targetSubnet string, knownHosts []string, nmapReport *gonma
 	 * summary to "Discovered Hosts" instead of "Unknown Hosts".
 	 */
 	if len(knownHosts) == 0 {
-		exposedVuln.Summary = "Discovered Hosts"
-		exposedVuln.Description = "At least one host has been discovered in the network"
+		exposedVuln.Summary = "Discovered Host"
+		exposedVuln.Description = "A host has been discovered in the network"
```

Comment:
In this case I think it shouldn't be a big deal because probably there won't be open issues but, in the case they were, the issue is determined by the `Summary` and `Description` fields, so if you change those a new issue would be created. Therefore we would loose the real age of the associated findings.

If you want we can check together if there are open issues in the Vulnerability DB or not.